findemailaddress2.py

```diff
@@ -0,0 +1,37 @@
+from bs4 import BeautifulSoup
+import requests
+import requests.exceptions
+import re
+import urllib
+
+
+emails = []
+website = requests.get("http://www.mit.edu/")
+
+
+
+#find links
+soup = BeautifulSoup(website.text,"html.parser")
+
+links = soup.find_all("a")
+
+
+
+for link in links:
+	link = link.get('href')
+	print ("processing " + link)
+	try:
+        	webpagedata = requests.get(link)
```

Comment:
wouldn't this eventually crawl the whole internet? say if mit.edu had a link to twitter.com?